spec/system/stackmap_spec.rb

```diff
@@ -12,19 +12,15 @@
     end
 
     it 'has a link to the stackmap on the record page' do
-      pending('Vendor fixing Stackmap javascript')
       visit '/catalog/99125428126306421'
-      expect(page).to have_button('Map it', wait: 5)
-      click_button 'Map it'
+      expect(page).to have_button('Where to find it', wait: 5)
+      click_button 'Where to find it'
```

Comment:
@sandbergja if you remove line 16 , will click_button wait by default? I would think it will wait if it is a button selector.

Reply:
I think it will wait until the default Capybara wait time, which I think is less than 5 seconds

Reply:
It's passing locally without line 16.  I vaguely remember some issue where it failed on CI though?  Pushing a separate branch without line 16 to check.

Reply:
It worked!  So I removed that line.